sources/packages/backend/apps/workers/src/controllers/assessment/assessment.controller.ts

```diff
@@ -469,6 +476,7 @@ export class AssessmentController {
    */
   private transformToAssessmentDTO(
     assessment: StudentAssessment,
+    noaApproval?: boolean,
```

Comment:
`noaApproval` is going to exist or not, I am not sure if there is a scenario where the nullable would be applicable.
Can we make it mandatory?